test/Fail/Issue723.err

```diff
@@ -5,15 +5,15 @@ Failed to solve the following constraints:
     create Bool : Default Set
     create zero : Default ℕ
     create defNat : Default (Default ℕ)
-    λ {ℓ} {A} → create (λ x → x) :
+    λ {_} {_} → create (λ x → x) :
```

Comment:
Shouldn't the names be there?  
If the names are not there, maybe it makes more sense to drop the whole hidden lambda?